recipe/meta.yaml

```
@@ -1,71 +1,74 @@
-{% set version = "1.14.1" %}
 {% set name = "geoviews" %}
+{% set version = "1.15.0" %}
 
 package:
   name: {{ name }}-split
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7f07f997eb19aedb414d2afe780744321033154a36cb46788913097d9912fff9
+  sha256: 07481e7346d33416857e1d7a41d6cd6a8261b62314d73c7edc9e45e612fa3ac9
 
 build:
   number: 0
-  #skip s390x as lack of nodejs and cartopy dependencies
-  skip: True  # [py<310 or s390x]
+  skip: true  # [py<310]
+
 requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
+    - hatchling
+    - hatch-vcs
+    - bokeh ==3.8.*
+
 outputs:
-  - name: {{ name }}-core
-    script: build_base.sh # [not win]
-    script: build_base.bat # [win]
+  - name: geoviews-core
+    script: build_base.sh  # [not win]
+    script: build_base.bat  # [win]
     build:
-      skip: True  # [py<310 or s390x]
+      skip: true  # [py<310]
       entry_points:
         - geoviews = geoviews.__main__:main
```

https://github.com/holoviz/geoviews/blob/v1.15.0/pyproject.toml#L73


```suggestion
        - geoviews = geoviews.command:main
```